### ISSUE TITLE
app.adjust.com

### ISSUE THREAD

**Issue by terrorist96** [EnergizedProtection/block#313]
<h1 align="center">Energized Protection - block ⚡</h1>
<div align="center">
  <strong>Let's make an annoyance free, better open internet, altogether!</strong>
</div>

<h2 align="center">Issue Submit Form</h2>
<div align="center">
  Provide the following infos properly, which will help us to resolve your issue quickly.
</div>

### Issue(s):
Type `x` in between `[ ]` and make sure there isn't any space between brackets. Example; for Your Selected Issue(s), type like this - `[x]` 
You can select more than one category of issues if you need to!

- [x] Whitelist
- [ ] Blacklist
- [ ] App Broken
- [ ] Website Broken
- [ ] Request
- [ ] Other Issue

<hr>

### Pack(s):
Write the name of the pack(s) you are using.

- Pack(s) Name: blu

<hr>

### Extension(s):
Also name the extension(s) if you are using any. If you don't then leave it blank.

- Extension(s) Name: 

<hr>

### Domain(s):
If you are submitting this issue for whitelist/blacklist issue, send us the domain(s) for whitelisting/blacklisting here. Kindly use the **Code Tag** to prevent tracking.

- Domain(s):

```
app.adjust.com
```

<hr>

### Your Config:
Just to ensure there is no issues or conflicts with other app/software/magisk module/extension/source list. 
Make sure you are running Energized Protection Service only.

- Client: 

- Version: 

<hr>

### Details:
Write us a lil bit more about your issue or query. You can attach any screen shot or log of the issue or advert, this will help to highlight it.

- Your Issue Detail: 
Breaks Spotify sign up

<hr>

<div align="center">
  <h2>Thank you for making Energized Protection great with your kind help!</h2>
</div>

<p align="center"><sub>A project by <a href="https://nayemador.com" target="_blank">Ador</a> with ❤<p>

<p align="center"><a href="https://saythanks.io/to/AdroitAdorKhan" target="_blank"><img src="https://img.shields.io/badge/Say%20Thanks-!-1EAEDB.svg?longCache=true&style=flat-square"></a><p>

**Comment by kah0922** [comment]
[Adjust is a mobile analytics company, and a lot of apps use it, so ublocking it would be detrimental to user privacy.](https://www.adjust.com/product/mobile-app-analytics/)

Also I was able to sign up with no issues via the app with the Blu List enabled (via AdGuard). Are you sure that's the issue?

**Comment by terrorist96** [comment]
Yes. Here's the full link: 
`https://app.adjust.com/6ypk9q?deep_link=spotify%3A%2F%2Fplaylist%2F37i9dQZF1EgxFxDfZ8b44f&campaign=&adgroup=&creative=&label=sp_cid%3Ac7511a95720eb193786519b694c8de0e&tracker_limit=200000&redirect=http%3A%2F%2Fopen.spotify.com%2Fplaylist%2F37i9dQZF1EgxFxDfZ8b44f%3F%3Fnd%3D1`

**Comment by kah0922** [comment]
What software are you using? I'm seeing no link like that on AdGuard, and that was after using a burner email to create another Spotify account.

**Comment by terrorist96** [comment]
Blokada. Maybe it has to do with having a family plan because that link is to go to the family mix playlist.

**Comment by kah0922** [comment]
Yeah, that's a redirect link.  I don't actually use Spotify, so I can't really test it for myself on how to access it.

It still would be a horrible idea to unblock `app.adjust.com` though. You'd be better off temporarily unblocking `app.adjust.com` rather than completely whitelisting it.

**Comment by terrorist96** [comment]
Blocking that domain also breaks the "report explicit content" function in the Spotify app.

**Comment by AdroitAdorKhan** [comment]
@terrorist96 I really can't whitelist this pure analytics and tracking domain. You can whitelist this domain temporarily, and do your work, then block it again.

**Comment by terrorist96** [comment]
No problem

**Comment by Poopooracoocoo** [comment]
Another issue is that you cannot view Spotify playlists from open.spotify.com links. On my device with Blokada, Spotify links will fail when they redirect to app.adjust.com. 

When Blokada is off, Spotify forces itself to open the app. In browsers other than Chrome or in incognito tabs, you can get the Spotify website to open by telling it that you don't have the app. Spotify is so annoying. 

Here's a Spotify playlist you can test out: https://open.spotify.com/playlist/6PmSUbLMNvfv7oOiXrhXrH?si=91B9HHR7Tc2ug2VqmAL_oA
